Dockerfile

```diff
@@ -5,7 +5,8 @@ FROM ubuntu:20.04
 ENV DEBIAN_FRONTEND=noninteractive
 
 # Apt update
-RUN apt-get update -y
+RUN apt-get update -y &&  apt-get install --fix-missing
```

Comment:
This flags reinstalls missing packages? in what case would we have a missing file or package, maybe a bad configuration from our dependencies?

Reply:
No, it looks for packages in different IP address so it can pull all of them